_health-care/_js/_form.js

```diff
@@ -1250,10 +1297,14 @@ var HealthApp = (function() {
     }
 
     // Set up auto-save on any changes to the form.
-    getFormRoot().addEventListener("change",
+    formRoot.addEventListener("change",
         function (e) {
           saveForm(formRoot, e);
         });
+
+    $(allInputs).on('blur', function() {
+      validate(this);
```

Comment:
validating on blur versus change....which makes more sense? (I don't understand the differences here)